runtests.sh

```diff
@@ -0,0 +1,11 @@
+
```

Comment:
i guess adding a copy config.dist.yml to config.yml would solve the run issue on ci build. Probably best to add a config.ci.yml which already has proper setup for it.

Reply:
Okay, I can see how in [bootstrap.php we load in the yaml config](https://github.com/thegreenwebfoundation/thegreenwebfoundation/blob/ca-set-up-ci/packages/greencheck/tests/bootstrap.php#L52-L53) with:

```php
$yamlconfig = $yaml->parse(file_get_contents($rootDir .'/tests/config.yml'));
```

It's not clear to me what the idiomatic way to load the right config in PHP land.

Because you can set environment variables easily in a `docker-compose.yml` file, I figure setting an env var then (maybe path to the config file), then in the code checking for the existence of this env var, otherwise defaulting back to `config.yml`.

This would let us use the same code in prod and CI, and dev.

👍 ?

Reply:
maybe this might help? https://symfony.com/doc/current/components/dotenv.html

over the last years symfony changed to using env vars instead of yaml configs, so i'm fine to move in the same way.

Reply:
It would take more time to integrate it then simply adding a config dist file with configs for the ci env and copying it over to config.yml right before we run the tests.

Reply:
Okay, given we're already using the config file approach, I prefer your suggestion keeping the config.yml file but adding a check for an overridden path from the environment to a ci specific config,.yml file, and defaulting to the usual config.yml location of nothing is there.

We're already using it, and it reduces the number of new ideas to grok in the codebase. I'll push something tomorrow with this.